sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/AccessPointAdapterTest.java

```diff
@@ -104,10 +104,10 @@ void checkFCDALimitations_should_fail_with_four_error_messages() throws Exceptio
         //Then
         assertThat(sclReportItems).hasSize(4)
                 .extracting(SclReportItem::getMessage)
-                .containsExactlyInAnyOrder("There are too much FCDA for the DataSet DATASET3 for the LDevice LD_INST21 in IED IED_NAME",
-                        "There are too much FCDA for the DataSet DATASET6 for the LDevice LD_INST21 in IED IED_NAME",
-                        "There are too much FCDA for the DataSet DATASET6 for the LDevice LD_INST22 in IED IED_NAME",
-                        "There are too much FCDA for the DataSet DATASET5 for the LDevice LD_INST22 in IED IED_NAME");
+                .containsExactlyInAnyOrder("There are too much FCDA for the DataSet DATASET3 for the LDevice LD_INST21 in IED IED_NAME : 2 > (max) 1",
```

Comment:
Update issue or roolback changes

Reply:
I updated the issue in Jira and Github for all error messages.